client/my-sites/menus/item-options/posts.jsx

```diff
@@ -54,6 +54,7 @@ var Posts = React.createClass( {
 		var posts = this.props.posts.slice();
 		if ( this.props.type === 'page' ) {
 			posts.unshift( siteMenus.generateHomePageMenuItem( this.getHomePageTitle() ) );
+			posts.push( siteMenus.generateNewPageMenuItem() );
```

Comment:
I would suggest having this as the first or second item. If a person has a lot of pages, they won't know this option exists.